view/frontend/web/js/pargo.js

```diff
@@ -8,668 +8,626 @@
  * @author     dev@pargo.co.za
  */
 require([
-  "jquery",
-  "domReady!",
-  "ko",
-  "uiComponent",
-  "Magento_Checkout/js/model/quote",
-  "Magento_Checkout/js/checkout-data",
-  "Magento_Customer/js/customer-data",
-  "Magento_Checkout/js/checkout-data",
-  "Magento_Checkout/js/action/select-shipping-address",
-  "Magento_Checkout/js/action/create-shipping-address",
-  "Magento_Checkout/js/action/create-billing-address",
+    "jquery",
+    "domReady!",
+    "ko",
+    "uiComponent",
+    "Magento_Checkout/js/model/quote",
+    "Magento_Checkout/js/checkout-data",
+    "Magento_Customer/js/customer-data",
+    "Magento_Checkout/js/checkout-data",
+    "Magento_Checkout/js/action/select-shipping-address",
+    "Magento_Checkout/js/action/create-shipping-address",
+    "Magento_Checkout/js/action/create-billing-address",
 ], function (
-  $,
-  ko,
-  Component,
-  quote,
-  domReady,
-  checkout,
-  customer,
-  checkoutData,
-  selectShippingAddressAction,
-  createShippingAddress,
-  createBillingAddress
+    $,
+    ko,
+    Component,
+    quote,
+    domReady,
+    checkout,
+    customer,
+    checkoutData,
+    selectShippingAddressAction,
+    createShippingAddress,
+    createBillingAddress
 ) {
-  (function (win) {
-    "use strict";
-    var listeners = [],
-      doc = win.document,
-      MutationObserver = win.MutationObserver || win.WebKitMutationObserver,
-      observer;
-
-    function ready(selector, fn) {
-      // Store the selector and callback to be monitored
-      listeners.push({
-        selector: selector,
-        fn: fn,
-      });
-      if (!observer) {
-        // Watch for changes in the document
-        observer = new MutationObserver(check);
-        observer.observe(doc.documentElement, {
-          childList: true,
-          subtree: true,
-        });
-      }
-      // Check if the element is currently in the DOM
-      check();
-    }
+    (function (win) {
+        "use strict";
+        let listeners = [];
+        let doc = win.document;
+        let MutationObserver = win.MutationObserver || win.WebKitMutationObserver;
```

Comment:
Not sure about this one, but it might be fine as a const as well I think., No biggy.